core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -736,7 +736,7 @@ public void testRenameTableMissingSourceTable() {
     AssertHelpers.assertThrows(
         "Should reject renaming a table that does not exist",
         NoSuchTableException.class,
-        "Table does not exist",
+        "does not exist",
```

Comment:
Why was this needed?

Reply:
I've  updated the error msg so that we don't need that change